### DISCUSSION TITLE
Margin affects the theme.

### DISCUSSION THREAD

**Discussion by seanmrnda** [HollowMan6/mdbook-pdf#52] [Ideas]
![image](https://github.com/user-attachments/assets/082baffc-25fe-4668-a013-1c6260fc8f60)

My PDF theme currently extends to the edges of the page. When I add top and bottom margins, the theme gets cut off. Is there a way to add margins without affecting the theme in the PDF output?

Here's how it looks like with margin top-bottom:
![image](https://github.com/user-attachments/assets/01a63e7b-671c-44e2-a895-85b7e25a11fa)

Config with margin: 
![image](https://github.com/user-attachments/assets/c42acc9e-8947-403d-bc5a-063fb08e4699)

**Comment by HollowMan6** [comment]
Then you will need to override some theme styling by moving the margin specifications in the CSS files (i.e. css/print.css, which is the style for printer output) instead of in the `book.toml`. Please refer to the mdBook official documentation: https://rust-lang.github.io/mdBook/format/theme/index.html?highlight=css%20print#theme

Although Chrome determines the PDF rendering logic, the current behavior is reasonable IMO and consistent when printing a picture: the margins are always white. As there's no way to fill in those margins unless GenAI helps here.
